PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java

```diff
@@ -22,6 +22,19 @@
      */
     
     public static void main(String[] args) {
+        PersonalTwitterFeed feed = new PersonalTwitterFeed();
+        feed.newTweet();
```

Comment:
Pretty much the same thing I did but instead you pushed all the code to new Tweet for the tweet and i left the dialogue in main and moved the loop to new tweet and called on it instead of the whole thing.